saleor/graphql/menu/mutations.py

```diff
@@ -7,16 +7,33 @@
 from graphql_relay import from_global_id
 
 from ...menu import models
+from ...menu.error_codes import MenuErrorCode
 from ...menu.utils import update_menu
 from ...page import models as page_models
 from ...product import models as product_models
 from ..core.mutations import BaseMutation, ModelDeleteMutation, ModelMutation
+from ..core.types.common import MenuError
 from ..page.types import Page
 from ..product.types import Category, Collection
 from .enums import NavigationType
 from .types import Menu, MenuItem, MenuItemMoveInput
 
 
+class MenuErrorMixin:
```

Comment:
Do we really want to declare and maintain base mutation for each module?
It seems scary for me. 
Why don't we use `__init_subclass_with_meta__` in BaseMutation to dynamically declare expected fields for the given class?.
```
class MenuUpdate(ModelMutation):
    class Meta:
        ERROR_TYPE_CLASS = MenuError
        ERROR_TYPE_FIELD = "menu_errors"
        error_type = MenuError
```
`__init_subclass_with_meta__` could use this fields to generate error structure. This will add some magic to base mutations but even though I think that this is better than declaring new base mutation per module.

Reply:
The last commit brings that implementation. ;)